hoomd/hpmc/update.py

```diff
@@ -280,15 +280,16 @@ def perturb(timestep):
         return self.cpp_updater.getTotalCount(mode);
 
 
-class muvt(_updater):
+class MuVT(Updater):
     R""" Insert and remove particles in the muVT ensemble.
 
     Args:
-        mc (:py:mod:`hoomd.hpmc.integrate`): MC integrator.
         seed (int): The seed of the pseudo-random number generator (Needs to be the same across partitions of the same Gibbs ensemble)
-        period (int): Number of timesteps between histogram evaluations.
-        transfer_types (list): List of type names that are being transferred from/to the reservoir or between boxes (if *None*, all types)
+        trigger (int): Number of timesteps between grand canonical insertions
+        transfer_types (list): List of type names that are being transferred from/to the reservoir or between boxes
```

Comment:
Arguments that are also attributes should be listed again in an `Attributes:` doc section at the end of the class docstring.

Reply:
adding partial documentation for attributes

[ ] make sure `update.muvt()` works, expose C++ setter/getters as attributes, and document